src/components/modify-todo-box.tsx

```diff
@@ -0,0 +1,97 @@
+import { SyntheticEvent, useEffect, useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import useTodos from "@/hooks/useTodos";
+import { Button } from "@/components/ui/button";
+import { Card } from "@/components/ui/card";
+import { Label } from "@/components/ui/label";
+import { Textarea } from "@/components/ui/textarea";
+import { Input } from "@/components/ui/input";
+import { Todo } from "@/types/todo-type";
+
+const ModifyTodoBox = () => {
+  const params = useParams();
+  const navigate = useNavigate();
+  const [todo, setTodo] = useState<Todo | null>(null);
+  const { isLoading, getTodoById, modifyTodo, deleteTodo } = useTodos();
+
+  const handleUpdate = (e: SyntheticEvent) => {
+    e.preventDefault();
+
+    const form = e.target as HTMLFormElement;
+    const titleInput = form.elements.namedItem("title") as HTMLInputElement;
+    const descInput = form.elements.namedItem(
+      "description"
+    ) as HTMLInputElement;
+
+    if (e.target) {
+      const newTodo: Omit<Todo, "id" | "createdAt" | "completed"> = {
+        title: titleInput.value ?? "",
+        description: descInput.value ?? "",
+      };
+
+      modifyTodo(params.id!, newTodo);
```

Comment:
non-null assertion 연산자 없이, 작성할 수도 있을까요? 앞쪽에서 id 조건확인해주면 될 것 같아요.
```suggestion
      modifyTodo(params.id, newTodo);
```

Reply:
[73483b0](https://github.com/f-lab-edu/benee-todo/pull/2/commits/73483b070daa7bdddc2226ff947314766ae18e87) 에서 수정했습니다.
params.id에 관련된 부분 전반을 통일해 수정해보았습니다!